includes/class-wc-payments-payment-method.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class WC_Payments_Payment_Method
+ *
+ * @package WooCommerce\Payments
+ */
+
+use Automattic\WooCommerce\Blocks\Payments\Integrations\AbstractPaymentMethodType;
+
+/**
+ * The payment method, which allows the gateway to work with WooCommerce Blocks.
+ */
+class WC_Payments_Payment_Method extends AbstractPaymentMethodType {
```

Comment:
This class name should probably mention Blocks, no? Otherwise it might be unclear to maintainers that this is Blocks specific code.

Reply:
I was following the naming scheme from the blocks repo, but your suggestion makes sense. I renamed the class in https://github.com/Automattic/woocommerce-payments/pull/723/commits/ec79d0a1a0343e8db1a160656b7534c60fbcb563.